app/src/interfaces/list-m2m/list-m2m.vue

```diff
@@ -609,6 +641,13 @@ const { createAllowed, updateAllowed, deleteAllowed, selectAllowed } = useRelati
 			multiple
 			@input="select"
 		/>
+
+		<drawer-batch
+			v-model:active="batchEditActive"
+			:primary-keys="relatedPrimaryKeys"
+			:collection="relationInfo.relatedCollection.collection"
+			@refresh="batchRefresh"
+		/>
```

Comment:
The drawer-batch component will save the changes to the related items immediately, without staging them to the parent. This goes against how the rest of the app data handling works, where all changes are staged nested and only committed to the database on save of the root parent item. In order for this to work we should update the drawer batch component to return the changed values through an event on "save" instead of saving it to the API directly so this interface can set those changes like it would for 'single' nested updates

Reply:
Fair. I'll update asap.

Reply:
Updated with staging updates.

![batcheditingStageUpdates](https://github.com/directus/directus/assets/6641242/ef899fed-edfa-4391-bbb1-bf222ed75667)